targets/targets.json

```diff
@@ -336,7 +336,7 @@
         "default_lib": "small",
         "release_versions": ["2"]
     },
-    "LPC4088": {
+    "MCU_PC4088": {
```

Comment:
This should be `MCU_LPC4088`.

Also, you need to remove the `"release_versions": ["2", "5"]` line from `MCU_LPC4088`.